### PR TITLE
gRPC and SSE coverage for OpenTelemetry

### PR DESCRIPTION
**Pull request by rsvoboda**
gRPC and SSE coverage for OpenTelemetry

Partial port of the coverage from https://github.com/quarkus-qe/quarkus-test-suite/tree/main/monitoring/opentracing-reactive-grpc + small rework.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [ ] Refactoring
- [ ] Backport
- [x] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: monitoring/opentelemetry-reactive/src/test/java/io/quarkus/ts/opentelemetry/reactive/OpenTelemetryGrpcIT.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import io.quarkus.test.bootstrap.JaegerService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.JaegerContainer;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class OpenTelemetryGrpcIT {
+
+    @JaegerContainer(useOtlpCollector = true, image = "quay.io/jaegertracing/all-in-one:1.41.0")
```
Comment (rsvoboda) [inline monitoring/opentelemetry-reactive/src/test/java/io/quarkus/ts/opentelemetry/reactive/OpenTelemetryGrpcIT.java:23]: I guess I should drop `quay.io/jaegertracing/all-in-one:1.41.0` occurrences and update the default in framework.

Waiting till the review is done and CI run finished.

**Review by mjurc — APPROVED** [review]
Besides the call to be made, this can be merged. Thanks for update.

File: monitoring/opentelemetry-reactive/src/test/java/io/quarkus/ts/opentelemetry/reactive/OpenTelemetryGrpcIT.java (hunk 1 above)
Comment (rsvoboda) [reply monitoring/opentelemetry-reactive/src/test/java/io/quarkus/ts/opentelemetry/reactive/OpenTelemetryGrpcIT.java:23]: My PR worked with both versions, we can merge. I will handle version bump in the FW by the end of the week.

**Comment by rsvoboda** [comment]
`quay.io/jaegertracing/all-in-one:1.41.0` dropped

**Pull request merged**